### PR TITLE
pr1: Updated README.md

### PR DESCRIPTION
**Pull request by manikandan-kumaresan**
Added feature

### CODE REVIEW

**Comment by manikandan-kumaresan** [comment]
review comment1

**Comment by manikandan-kumaresan** [comment]
> Added feature

what feature

**Comment by syncuser123** [comment]
Code changes reviewed. No concerns

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # ODBC-Jira
 ODBC-Jira
 Change 1
+Feature added
```
Comment (syncuser123) [inline README.md:4]: What feature

**Review by syncuser123 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (manikandan-kumaresan) [reply README.md:4]: Test dummy feature1

**Pull request merged**